src/Transformer/Denormalization/DefaultTransformer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flagbit\Bundle\CategoryBundle\Transformer\Denormalization;
+
+use Flagbit\Bundle\CategoryBundle\Transformer\DenormalizationTransformer;
+
+/**
+ * Default denormalization transformer that applies no changes.
+ */
+class DefaultTransformer implements DenormalizationTransformer
+{
+    /**
+     * @phpstan-param mixed $data
+     *
+     * @return mixed
+     */
+    public function transform($data)
+    {
+        return $data;
+    }
+}
```

Comment:
I'm currently thinking about that the DefaultTransformer is actually a StringTransformer, because it's supposed to get a string from imported data.